configuration/configuration_reference.md

```diff
@@ -877,6 +857,211 @@ can be used to **customize how config repo plugin works when parsing this specif
 
 [top](#top)
 
+## &lt;svn&gt; {#config-repo-svn}
+
+The config repo `<svn>` material element specifies the location of your code base in Subversion repository.
+
+### Attributes
+
+| Attribute | Required | Description |
+|-----------|----------|-------------|
+| url | Yes | URL for the remote repository, for example: 'http://www.thoughtworks-studios.com/go-agile-release-management/svn/myproject/trunk'. Go supports the following protocols for subversion: http, https, svn and svn+ssh, but does not support 'file:///'. |
+| username | No | The user account for the remote repository. |
+| password | No | The password for the specified user |
+| encryptedPassword | No | The encrypted password for the specified user |
+| checkexternals | No | The default value is false, the value should be either one of true/false or 1/0. 'true' or '1' means that the changes of externals will trigger the pipeline automatically. |
+| materialName | Required if this material is referenced in pipeline labeltemplate | The name to identify a material. Material name can contain the following characters: a-z, A-Z, 0-9, fullstop, underscore and hyphen, but whitespace is not allowed. A material name is case insensitive and starting with fullstop is invalid. It needs to be unique within a pipeline. The max length is 255 characters. |
```

Comment:
`Required if this material is referenced in pipeline labeltemplate` - This doesn't belong here. There are no pipelines and hence no labelTemplate(s) coming into picture. Just say that its optional. 
Please remind me why we need a materialName field for this.

Reply:
@jyotisingh -- the material name is not necessary. It is an optional field. I don't think there is any reason for keeping it around.